src/porting.cpp

```diff
@@ -697,6 +705,28 @@ int mt_snprintf(char *buf, const size_t buf_size, const char *fmt, ...)
 	return c;
 }
 
+bool openURL(std::string url)
+{
+	if (!std::regex_match(url,
+			std::regex(R"(^https?:\/\/[\w\.-\/\?=\%\+\#\&]+$)") )) {
```

Comment:
Note that I still think the regex should be removed and replaced with a simple `begins with "http://" or begins with "https://"` check

Reply:
Yes, that would simplify things a lot. Although I wonder how far it should be restricted.

> Instead, I'd prefer to have a button to open up ContentDB in a web browser.

But with these restrictions it would not be possible to open a project page like
`https://content.minetest.net/packages/Calinou/moreblocks/`

Reply:
![image](https://user-images.githubusercontent.com/2122943/71845989-6c2e0c80-30c1-11ea-93df-d2ad997eb644.png)

Reply:
![image](https://user-images.githubusercontent.com/2122943/71846562-cd0a1480-30c2-11ea-8203-5d8bcb1aeb13.png)